src/modules/api.js

```diff
@@ -0,0 +1,42 @@
+export const gameUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/9W40RRKEzdVGseFky5fH/scores/';
+// this part for generating gameID
+// fetch(Url, {
+//   method: 'POST',
+//   body: JSON.stringify({
+//     name: 'GeekyHacks'
+//   }),
+//   headers: {
+//     'Content-type': 'application/json; charset=UTF-8',
+//   },
+// })
+//   .then((response) => response.json())
+//   .then((json) => console.log(json));
```

Comment:
- [ ] It is unprofessional to leave commented code in your codebase so I kindly suggest you remove these highlighted lines of code. 👍 ✨